### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by msgjt**

### CODE REVIEW

**Review by msgjt — COMMENTED** [review]
test

**Review by schwarad — APPROVED** [review]
ok

File: jbugs-business/src/main/java/edu/msg/jbugs/assemblers/UserAssembler.java [hunk 1]
```diff
@@ -19,38 +19,36 @@
 public class UserAssembler {
 
 	public UserAssembler() {
+		//default constructor
 	}
 
 	public UserEntity dtoToModel(UserDTO dto) {
```
Comment (schwarad) [inline jbugs-business/src/main/java/edu/msg/jbugs/assemblers/UserAssembler.java:25]: heh

**Review by schwarad — APPROVED** [review]
test

**Review by schwarad — APPROVED** [review]

**Comment by schwarad** [comment]
test

**Review by schwarad — APPROVED** [review]
test

**Review by schwarad — APPROVED** [review]
approved

**Review by schwarad — APPROVED** [review]

**Comment by schwarad** [comment]
approved

**Review by schwarad — APPROVED** [review]

**Pull request merged**